web/rootfs/defaults/settings-config.js

```diff
@@ -152,22 +155,20 @@ config.dropbox.redirectURI = '{{ .Env.DROPBOX_REDIRECT_URI }}';
 {{ end -}}
 {{ end -}}
 
-{{ if $ENABLE_FILE_RECORDING_SERVICE -}}
-// When integrations like dropbox are enabled only that will be shown,
-// by enabling fileRecordingsServiceEnabled, we show both the integrations
-// and the generic recording service (its configuration and storage type
-// depends on jibri configuration)
-config.fileRecordingsServiceEnabled = true;
-{{ end -}}
-{{ if $ENABLE_FILE_RECORDING_SERVICE_SHARING -}}
+{{ if $ENABLE_FILE_RECORDING_SHARING -}}
 // Whether to show the possibility to share file recording with other people
 // (e.g. meeting participants), based on the actual implementation
 // on the backend.
-config.fileRecordingsServiceSharingEnabled = true;
+config.recordingService.sharingEnabled = true;
 {{ end -}}
 {{ end -}}
 
 
+// Local recording configuration.
+if (!config.hasOwnProperty('localRecording')) config.localRecording = {};
+config.localRecording.disable = $DISABLE_LOCAL_RECORDING;
```

Comment:
Can you add variables for the rest of the settings, while you're here?

Reply:
These setting seems disabled by default.

What would be the environment variable names ?

```
DISABLE_LOCAL_RECORDING_NOTIFY_ALL_PARTICIPANT
DISABLE_LOCAL_RECORDING_SELF_RECORDING
```
or
```
ENABLE_LOCAL_RECORDING_NOTIFY_ALL_PARTICIPANT
ENABLE_LOCAL_RECORDING_SELF_RECORDING
```